pkg/arguments/arguments.go

```diff
@@ -335,9 +336,12 @@ func MarkRegionDeprecated(parentCmd *cobra.Command, childrenCmds []*cobra.Comman
 		cmd.Run = func(c *cobra.Command, args []string) {
 			outputFlag := cmd.Flag("output")
 			regionFlag := cmd.Flag("region")
+			disableDeprecationFlag := cmd.Flag(DisableRegionDeprecationFlagName)
 			hasChangedOutputFlag := outputFlag != nil && outputFlag.Value.String() != outputFlag.DefValue
```

Comment:
can this be changed to 
`			hasChangedOutputFlag := cmd.Flags().Changed("output")
			hasChangedRegionFlag := cmd.Flags().Changed("region")
			isRegionDeprecationDisabled := cmd.Flags().Changed(DisableRegionDeprecationFlagName)`

instead?

Reply:
Wasn't able to get this to work sadly, not sure why :/

Reply:
specifically with `region` - i think it has a default value which causes it to be changed even when it's not